plugin.xml

```diff
@@ -153,6 +153,8 @@ to config.xml in order for the application to find previously stored files.
         <source-file src="src/android/AssetFilesystem.java" target-dir="src/org/apache/cordova/file" />
         <source-file src="src/android/PendingRequests.java" target-dir="src/org/apache/cordova/file" />
 
+        <framework src="androidx.webkit:webkit:1.3.0" type="gradleReference" />
```

Comment:
Will this interfere with https://github.com/apache/cordova-android/blob/f100809bf386c9c53e0d1c1e8f3149517862a883/framework/build.gradle#L80 ?

Not sure if this line is really needed. Our current [default version](https://github.com/apache/cordova-android/blob/f100809bf386c9c53e0d1c1e8f3149517862a883/framework/cdv-gradle-config-defaults.json#L9) is `1.4.0`

Reply:
The default version is only applied to the CordovaLib and is not available to the app.

I could try 1.4.0 but, version 1.3.0 was taken from the plugin that used the proxy implementation originally. If bumped might break and force other users to update their plugin. Not sure what will happen if there are multiple versions set.

Reply:
https://github.com/apache/cordova-plugin-file/pull/513/commits/d11fe941db55d1d10fde7965d6fae6b3a65bfb5f